cli/cmd/service.go

```diff
@@ -2140,6 +2140,11 @@ func (c *ServicedCli) cmdServiceTune(ctx *cli.Context) {
 		return
 	}
 
+	if ctx.IsSet("help") {
```

Comment:
Can you verify that this is needed? Do we already handle `--help` elsewhere?

Reply:
Right you are, I'll remove it.